### DISCUSSION TITLE
Intel HD 3000 PC Hackintosh

### DISCUSSION THREAD

**Discussion by chris1111** [chris1111/Legacy-Video-patch#9] [General]
**For PC Hackintosh**
Even with the Legacy Video Patch if you not have DSDT patch HD 3000 you need to setup Clover config.plist like this

Under Graphics
![Graphics](https://user-images.githubusercontent.com/6248794/102701509-aab2cd80-4225-11eb-8a9c-14538b39144b.png)
Under FakeID
![FakeID](https://user-images.githubusercontent.com/6248794/102701511-af778180-4225-11eb-990a-f97cb75e370d.png)

**Comment by dopf-26** [comment]
What is the corresponding alternative for opencore?

**Reply by chris1111** [reply]
> What is the corresponding alternative for opencore?

Using WhateverGreen.kext
Device Property
![Capture d’écran, le 2021-01-06 à 18 00 25](https://user-images.githubusercontent.com/6248794/103830540-31024880-5049-11eb-9416-e94345db96ef.png)

![Capture d’écran, le 2021-01-06 à 17 57 13](https://user-images.githubusercontent.com/6248794/103829458-e680cc00-5048-11eb-93e9-683580bf7ac2.png)

**Reply by dopf-26** [reply]
Thanks, you are a legend! Do I need to keep SIP disabled after installing?

**Reply by chris1111** [reply]
Yes

**Reply by chris1111** [reply]
> And on BigSur?


https://github.com/chris1111/Legacy-Video-patch/discussions/11

**Reply by jian-19** [reply]
hello, could you help me apply the legacy patch? has the hd 3000, I've reinstalled catalina more than 20 times, and whenever I install the patch, MacOS won't start...

**Reply by SwankySanam** [reply]
Bro which smbios i should go with for i3 2370m hd 3000 on catalina

**Reply by chris1111** [reply]
> i3 2370m



MacBook Pro 9.1 for laptop you need DSDT [patch HD 3000](https://github.com/RehabMan/Laptop-DSDT-Patch/blob/master/graphics/graphics_HD3K_low.txt)
 [for High Screen ](https://github.com/RehabMan/Laptop-DSDT-Patch/blob/master/graphics/graphics_HD3K_high.txt)

**Reply by SwankySanam** [reply]
> > i3 2370m
> 
> MacBook Pro 9.1 for laptop you need DSDT [patch HD 3000](https://github.com/RehabMan/Laptop-DSDT-Patch/blob/master/graphics/graphics_HD3K_low.txt)
> [for High Screen ](https://github.com/RehabMan/Laptop-DSDT-Patch/blob/master/graphics/graphics_HD3K_high.txt)

I hav generated a dsdt for it 
Hd 3000 works fine on Mojave with ur video patch 
Now its turn for catalina
Btw thanks for the reply 🙂

**Reply by LuckyLuck8** [reply]
After installing patch HD 3000 works but I have graphics artifacts on screen and freezes (screen freeze, system does not respond, only mouse cursor is moving)
using latest Clover + WEG
dev-id 0116 , snb-platform-id 00000200 (with 00000100 has the same behavior)
Also in 10.14.6 && 10.13.6 have the same freezes and artifacts
But in 10.12.6 HD 3000 works perfect, without any freezes and artifacts (with the same dsdt, kexts and clover)
p.s. maybe someone has MBP 8.1 or any other mac with HD 3000  and can share its original dsdt (or full acpi dump) and IOReg ?

**Reply by HassanElDessouki** [reply]
> > i3 2370m
> 
> MacBook Pro 9.1 for laptop you need DSDT [patch HD 3000](https://github.com/RehabMan/Laptop-DSDT-Patch/blob/master/graphics/graphics_HD3K_low.txt) [for High Screen](https://github.com/RehabMan/Laptop-DSDT-Patch/blob/master/graphics/graphics_HD3K_high.txt)

hello @chris1111 
thanks for your tools!
I had a question, how do I use the .txt file for DSDT patching? Do I create a *.aml file with the text in the file?
Please clarify.. and thanks!

**Comment by rmottatb** [comment]
Thank you for the patch, but I'm getting this error: "runtime script error apple event timed out" and I couldn't install, there is something else I have to do before make the app and install?

I tried with Catalina 15.15.4 and 15.5.7

**Reply by oldman20** [reply]
> Thank you for the patch, but I'm getting this error: "runtime script error apple event timed out" and I couldn't install, there is something else I have to do before make the app and install?
> 
> I tried with Catalina 15.15.4 and 15.5.7

same result!
